### DISCUSSION TITLE
RSS/Atom feed for Temurin releases?

### DISCUSSION THREAD

**Discussion by regunakyle** [adoptium/adoptium-support#1141] [Q&A]
I want to track Temurin releases with RSS, but I couldn't find a RSS/Atom feed for it. If it exists, can someone point me to it?

For reference, I am tracking Python and Node.js releases with the following Github feeds:
1. https://github.com/python/cpython/tags.atom
2. https://github.com/nodejs/node/releases.atom

**Comment by karianna** [comment]
@johnoliver - Is that something the API could produce?
